src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -645,14 +645,21 @@ class TezosNodeOperator(
     val proposalsStateFetch =
       fetchMerge(currentQuorumFetcher, currentProposalFetcher)(CurrentVotes.apply)
 
-    def parseMichelsonScripts: Block => Block = { block: Block =>
+    def parseMichelsonScripts: Block => Block = {
       implicit lazy val _ = logger
+      def rewriteParametersInTransactions(transaction: Transaction): Transaction = {
+        val expr = transaction.parameters.map {
+          case Left(value) => value.value.expression
```

Comment:
I'd suggest pattern matching on `value` too, to help when reading the code

Reply:
ok, patter matched and renamed for clarity